src/govuk/components/accordion/accordion.mjs

```diff
@@ -36,6 +36,7 @@ var ACCORDION_TRANSLATIONS = {
  * @class
  * @param {HTMLElement} $module - HTML element to use for accordion
  * @param {AccordionConfig} [config] - Accordion config
+ * @this {Accordion}
```

Comment:
Yeah I think it came from a brief time I was trying to run in strict mode

```js
Accordion() // this === window
new Accordion() // this === instanceof Accordion
```

Where inside our constructor function we'd technically need to add:

```js
function Accordion ($module, config) {
  if (!(this instanceof Accordion)) {
    return new Accordion($module, config);
  }
}
```

☝️ Old trick to detects the missing `new` and returns an instance of itself

Shall I get rid? Won't need it once we switch to classes anyway

Reply:
I'd be keen to drop the `@this` then, yeah, to not carry its limited value 😄 

(not sure if you were proposing to introduce the check for use as a function rather than a constructor, but given that'd expand our API to allow function right before we'd contract it back by switching to classes, I don't feel it's necessary either).

Reply:
Nah too much overhead. Compiler was right but we can live with it for now

I've removed `@this` from JSDoc and updated the coding standards to match